lib/semian/net_http.rb

```diff
@@ -103,6 +108,8 @@ def with_resource_timeout(timeout)
       end
     end
 
+    attr_accessor :semian_enabled
```

Comment:
```suggestion
    attr_accessor :semian_enabled
```

Let's not allow to change it after the connection was created, because it opens a huge can of worm.

Reply:
👍